projects/quests.md

```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
+
+## Fund a quest
+
+**Patrons** can fund a Quest only when connected to a wallet. The Fund button is in the detail view of a Quest.
+
+The Patron should only submit the amount of funding for the Quest reward token.
+
+## Claim a Quest bounty
+
+**Players** can claim the bounty only when connected to a wallet. The **Schedule claim** button is in the detailed view of a Quest.
+
+The submitted claim will be delayed for a specific period where other users can challenge the claim if the requirements of the Quest aren't met (see [verification of Quest](quests.md#verifying-a-quest)).
+
+The Player msut submit the following in 2 steps :
+
+1. First:
+   - Evidence of completion (should meet the Quest description requirements)
+2. Second:
+   - The UI offers to claim all the Quest's bounty or just a specific amount (Player should pick what is specified in the Quest description)
+   - Player's wallet address to which the reward should be sent when a claim is completed
+3. The Player should deposit a collateral that will be restored once the claim is executed successfully.
+
+When the delay will be over, the Player can execute the claim by clicking on the **Claim** button next to the corresponding claim. (Note that anyone can execute the claim if the Quest isn't expired)
+
+## Challenge a claim
+
+Anyone can challenge a **Player**'s claim when connected to a wallet. (see [verification of Quest](quests.md#verifying-a-quest))
+The **Challenge** button is next to the Player claim.
+
+A challenger can only challenge during the delay period.
+
+The **Challenger** must submit the following :
+
+1. Reason why this claim should be challenged
+2. A deposit that will be restored when the challenge is resolved in favor of the Challenger
+
+This will then create a dispute in [Celeste](https://celeste.1hive.org/#/dashboard) (Note that with Rinkeby network, a dispute should be resolved manually by contacting us in the #Quests channel in the [1hive discord](https://discord.gg/NaCDVBs2))
+
+Once the dispute is resolved in Celeste, a stackholder of the dispute needs to click on the **Resolve** button next to the claim to spread the challenge's result to the app.
+
+If the dispute is resolved ... :
+
+- in favor of the **Player** : The claim will immediatly be executed and **Player** wins the **Challenger** deposit
+- in favor of the **Challenger** : The claim is rejected and the **Challenger** wins the **Player** deposit
+
+## Reclaim
+
+The **Creator** can reclaim remaining funds and deposit them when connected to the corresponding wallet. This **Reclaim** button is only in the detailed view of a Quest once the Quest is in the **Expired** state.
+
+If there are pending claims after the expiration date, the **Creator** should't reclaim the funds unless he doesn't want the claiming Players to collect their reward. The creator can execute the claims by himself once the Quest is expired.
```

Comment:
What is Pending claims?

Reply:
Claims that has not been rejected or executed (delay or waiting to execute)